packages/lumx-react/src/components/link/Link.tsx

```diff
@@ -32,9 +35,13 @@ const CLASSNAME: string = getRootClassName(COMPONENT_NAME);
  *
  * @return The component.
  */
-const Link: React.FC<LinkProps> = ({ children, className, color, colorVariant, ...props }) => {
+const Link: React.FC<LinkProps> = ({ children, className, linkRef, color, colorVariant, ...props }) => {
     return (
-        <a className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, color, colorVariant }))} {...props}>
+        <a
+            className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, color, colorVariant }))}
+            ref={linkRef}
+            {...props}
```

Comment:
Why not keeping the ref in the other props?

Reply:
Functional component cannot forward ref while forwarding props.
One way you can forward ref is to use `React.fowardRef` but it adds a wrapper around your functional component and for LumX we decided to use explicit name for Ref props everywhere.

Reply:
Good to know. 
Even if the `a` is primary element returned?